### DISCUSSION TITLE
Should we change our licensing format?

### DISCUSSION THREAD

**Discussion by vmesel** [moclojer/moclojer#127] [Ideas]
Nowadays, inside the LICENSE.md file, we can see that Moclojer's rights are under Avelino's name, should we change it for the organization Moclojer in the near future?

If yes, what should be contained in the license.

**Comment by matheusfrancisco** [comment]
@avelino we can turn the LICENSE under Cljazz name's?

**Reply by avelino** [reply]
yep, send PR here https://github.com/moclojer/moclojer/pull/138
